avocado/utils/software_manager/backends/rpm.py

```diff
@@ -6,8 +6,7 @@
 from ... import process
 from .base import BaseBackend
 
-
-log = logging.getLogger('avocado.test')
+LOG = logging.getLogger('avocado.test')
```

Comment:
Changes look good, but I don't understand why this is separate from the previous commit.

Reply:
The first one I changed only the namespace, to keep consistent with others. The second one I fixed the usage of 'logging' , since it was bug it self (some parts where using LOG and others logging). I could have replaced s/log/LOG/ on the first yes, But I would still prefer to keep both commits.

Reply:
OK, fair enough.